crypto/src/lib.rs

```diff
@@ -1,13 +1,49 @@
+//! Cryptography dispatch for the edhoc-rs crate
```

Comment:
It can't: The whole point of crypto-trait is that it will not depend on any concrete back-ends.

Eventually this should only be needed for things like tests where you want to switch around back-ends by the flick of a feature (as it makes for easy test matrices). It may make sense to rename that crate to `edhoc-crypto-preferred-impl` -- but that crate would not make it to crates.io as long as it depends on a wide variety of not-on-crates optional dependencies.